### PR TITLE
Fix the onnx import error

### PR DESCRIPTION
**Pull request by yiweiliu008**
Summary:
The ExportTypes was removed from torch.onnx, but it is still referenced in code. This leads to error
```
ImportError: cannot import name 'ExportTypes' from 'torch.onnx'
```

This change add back the correct the reference to ExportTypes

Test Plan: Test run downstream code that has dependency on pytext.utils.onnx and see the error is gone.

Differential Revision: D64340520

### CODE REVIEW

**Review by malfet — COMMENTED** [review]
Public PR description is somewhat confusing. And why you are rename exporter_states.py?

**Comment by justinchuby** [comment]
I see the code from https://pytext.readthedocs.io/en/master/_modules/pytext/utils/onnx.html mentions caffe2, but support for caffe2 has been deprecated and is being removed if I am not mistaken. Could you share more on how the option is used?

File: torch/onnx/exporter_states.py
Comment (justinchuby) [inline torch/onnx/exporter_states.py:1]: File should be removed

**Comment by yiweiliu008** [comment]
> Public PR description is somewhat confusing. And why you are rename exporter_states.py?

I learnt that in Python, the underscore _ in a file name has a special meaning. It is used to indicate that the file is a "private" module, which means that it is not intended to be imported directly by other modules. I need to import the file therefore removed the "_" from the file name.

File: torch/onnx/exporter_states.py
Comment (yiweiliu008) [reply torch/onnx/exporter_states.py:1]: Could you please advice how to resolve import error?

**Comment by yiweiliu008** [comment]
> I see the code from https://pytext.readthedocs.io/en/master/_modules/pytext/utils/onnx.html mentions caffe2, but support for caffe2 has been deprecated and is being removed if I am not mistaken. Could you share more on how the option is used?

I'm not using this option but the current pytext/utils/onnx.py file is reporting error because this piece of code "export_type=ExportTypes.ZIP_ARCHIVE" in the file. I just want to resolve the error.

**Comment by malfet** [comment]
> I learnt that in Python, the underscore _ in a file name has a special meaning. It is used to indicate that the file is a "private" module, which means that it is not intended to be imported directly by other modules. I need to import the file therefore removed the "_" from the file name.

I suspect private namespace is very much by design, so please don't make it public without consulting with the module maintainers.

**Review by malfet — CHANGES_REQUESTED** [review]
Looks like `ExportTypes` is aimed for deprecation and should not be used

**Pull request closed without merging**